config/default/manager_auth_proxy_patch.yaml

```diff
@@ -11,6 +11,11 @@ spec:
       containers:
       - name: kube-rbac-proxy
         image: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.7.0
+        securityContext:
```

Comment:
minor thing - I guess snyk ignores the fact that this file is a patch.

I think updating this section should suffice: https://github.com/codeready-toolchain/member-operator/blob/ec800b33605d265754c7791b6345cc229f62a27e/config/manager/manager.yaml#L27-L28
with:
```
      securityContext:
        runAsNonRoot: true
        allowPrivilegeEscalation: false
        capabilities:
            drop:
               - all
```
so that we define it only once and future containers/patches will inherit it.

Reply:
that would be nice indeed, but `allowPrivilegeEscalation` and `capabilities` can only be set at container level.

Reply:
haa my bad 🤦‍♂️ , indeed seems like those 2 fields are available in the [SecurityContext](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#securitycontext-v1-core) struct but not in  [PodSecurityContext](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#podsecuritycontext-v1-core). While `runAsNonRoot` and others are present in both structs. Never mind , you do need both sections.